Dockerfile

```
@@ -105,23 +105,15 @@ ENV PATH=/opt/odoo-venv/bin:$PATH
 
 ARG odoo_version
 
-# Install Odoo requirements (use ADD for correct layer caching).
-# We use requirements from OCB for easier maintenance of older versions.
-# We use no-binary for psycopg2 because its binary wheels are sometimes broken
-# and not very portable.
-ADD https://raw.githubusercontent.com/OCA/OCB/$odoo_version/requirements.txt /tmp/ocb-requirements.txt
-RUN pip install --no-cache-dir --no-binary psycopg2 -r /tmp/ocb-requirements.txt
-
-# Install other test requirements.
-# - coverage
-# - websocket-client is required for Odoo browser tests
-RUN pip install --no-cache-dir \
-  coverage \
-  websocket-client
-
-# Install Odoo (use ADD for correct layer caching)
+# Use ADD to rebuild from here when Odoo HEAD changes
 ARG odoo_org_repo=odoo/odoo
 ADD https://api.github.com/repos/$odoo_org_repo/git/refs/heads/$odoo_version /tmp/odoo-version.json
+
+# Install Odoo requirements.
+COPY requirements-${odoo_version}.txt /tmp/requirements.txt
+RUN pip install --no-cache-dir --no-binary psycopg2 -r /tmp/requirements.txt
```

This is already in the files, right?

```suggestion
RUN pip install --no-cache-dir -r /tmp/requirements.txt
```